### DISCUSSION TITLE
Welcome to Prom Discussions!

### DISCUSSION THREAD

**Discussion by NicholasG04** [NicholasG04/Prom#278] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NicholasG04** [comment, marked as answer]
gm wagmi

**Comment by jmontg03** [comment]
Attempting to get galaxy brain!

**Comment by JesseAtSZ** [comment]
Attempting to cheat! I am a liar and a fool!
